rotatedarray

```diff
@@ -0,0 +1,29 @@
+class Solution {
```

Comment:
Time and Space complexity?

Reply:
added 3rd solution and provided TC and sc in readme, i ll try to add it as  comments in program from next submission